cmd/proxy/cmd/proxy.go

```diff
@@ -305,7 +319,17 @@ func (c *ClusterChecker) Start() error {
 				// report that check was ok
 				checkOkCh <- struct{}{}
 			}
-			timerCh = time.NewTimer(cluster.DefaultProxyCheckInterval).C
+
+			if previousCheckIntervalSeconds != c.checkIntervalSeconds {
+				log.Infow("updated check interval seconds is ", "seconds", c.checkIntervalSeconds)
+				previousCheckIntervalSeconds = c.checkIntervalSeconds
+			}
+			if previousRequestTimeoutSeconds != c.requestTimeoutSeconds {
+				log.Infow("updated request timeout seconds is ", "seconds", c.requestTimeoutSeconds)
+				previousRequestTimeoutSeconds = c.requestTimeoutSeconds
+			}
```

Comment:
I'll also log a warning if proxyCheckInterval isn't smaller than proxyTimeoutInterval since the proxy will always fail.